app/workers/deletes.php

```diff
@@ -425,7 +425,7 @@ protected function deleteFunction(Document $document, string $projectId): void
          * Delete Deployments
          */
         Console::info("Deleting deployments for function " . $functionId);
-        $storageFunctions = new Local(APP_STORAGE_FUNCTIONS . '/app-' . $projectId);
+        $storageFunctions = $this->getDevice(APP_STORAGE_FUNCTIONS . '/app-' . $projectId);
```

Comment:
Looks like there should already be a [`getFunctionsDevice()`](https://github.com/appwrite/appwrite/blob/02a432102df047d2ea96faed65d77a862239d487/src/Appwrite/Resque/Worker.php#L254) method on the Worker. Maybe that can be reused?

Reply:
Done